sklearn/cross_decomposition/pls_.py

```diff
@@ -367,8 +375,12 @@ def fit(self, X, Y):
             # Y = X W(P'W)^-1Q' + Err = XB + Err
             # => B = W*Q' (p x q)
             self.coef_ = np.dot(self.x_rotations_, self.y_loadings_.T)
-            self.coef_ = (1. / self.x_std_.reshape((p, 1)) * self.coef_ *
-                          self.y_std_)
+            # self.coef_ = (1. / self.x_std_.reshape((p, 1)) * self.coef_ *
+            #               self.y_std_)
```

Comment:
I am not familiar with PLS but if this is no longer needed please delete those commented out lines.